src/client/builtin_scene/meshes/loaders/ksplat_loader.hpp

```diff
@@ -0,0 +1,152 @@
+#pragma once
+
+#include <vector>
+#include <string>
+#include <memory>
+#include "../model_3d.hpp"
```

Comment:
This header file has been removed, please update the correct includes to use `builtin_scene::GaussianSplat`.